pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -349,7 +349,10 @@ def make_package(args):
     if args.icon_fg and args.icon_bg:
         shutil.copy(args.icon_fg, join(res_dir, 'mipmap/icon_foreground.png'))
         shutil.copy(args.icon_bg, join(res_dir, 'mipmap/icon_background.png'))
-        with open(join(res_dir, 'mipmap-anydpi-v26/icon.xml'), "w") as fd:
+        mipmap_anydpi = join(res_dir, 'mipmap-anydpi-v26')
+        if not exists(mipmap_anydpi):
+            os.mkdir(mipmap_anydpi)
+        with open(join(mipmap_anydpi, 'icon.xml'), "w") as fd:
```

Comment:
There is already defined function for checking path existence
```python3
        anydpi_dir = join(res_dir, 'mipmap-anydpi-v26')
        ensure_dir(anydpi_dir)
        with open(join(anydpi_dir, 'icon.xml'), "w") as fd:
```

Reply:
@danigm would you like to update the PR to use the `ensure_dir` utility function?

Reply:
I've added a new commit to use `ensure_dir`.